Tasks/DownloadPackageV0/task.loc.json

```diff
@@ -82,7 +82,7 @@
         "feed": "$(feed)",
         "definition": "$(definition)"
       },
-      "endpointUrl": "{{endpoint.url}}/{{ #regex ([a-fA-F0-9\\-]*/)[a-fA-F0-9\\-]+ feed }}_apis/Packaging/Feeds/{{ #regex [a-fA-F0-9\\-]*/([a-fA-F0-9\\-]+) feed }}{{#if view}}@{{{view}}}{{/if}}/Packages/{{{definition}}}/Versions?includeUrls=false&protocolType={{{packageType}}}",
```

Comment:
Why are we removing the protocolType in this? Won't it break existing users who use this field?

Reply:
It is not needed in the v0 version of the task since it only supports nuget anyway. I added it back to not change too many things in this PR.